src/assets/translations/en/main.json

```
@@ -1900,6 +1900,17 @@
         "button": "Pair"
       }
     },
+    "walletConnect": {
+      "topSection": {
+        "dontSeeWallet": "Don't see your wallet?",
```

Biome (2.1.2)

[error] The key walletConnect was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)